src/pages/agenda/[day].tsx

```diff
@@ -37,6 +41,10 @@ const Agenda: NextPage = () => {
     return <Error statusCode={404} />;
   }
 
+  function onChange(option: any) {
```

Comment:
Typically using `any` is a ~~bad thing~~ but the typing here would be annoying so this is fine.